### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/2182
https://github.com/BetaMasaheft/Documentation/issues/2181

### CODE REVIEW

File: new/LIT6702MashafaKobor.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6702MashafaKobor" passive="LIT5048Hassabat"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BDLaethe32"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>ለእመ፡ ኮነ፡ ኮቦር፡ ዮሐንስ፡ በሰኑይ፡ አልቦ፡ ደዌ፡ ለጸብእ፡ አገሳ፡ የሐልቁ፡ ዝናም፡ የሐፅፅ፡ መዓር፡ ይውኅዝ፡
+                        ሰገም፡ ይሰምር፡ ሰናይ፡ ይሰምር፡ ሕፃናተ፡ ይሙውቱ ፨</ab>
```
Comment (MassimoVilla) [inline new/LIT6702MashafaKobor.xml:63]: The handwriting is not very clear, but I read the nominative form ሕፃናት (which agrees with the following verb ይመውቱ፡). The vowel marker of ት is just hinted.

File: new/LIT6703HymnMik.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            <change who="DR" when="2022-08-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following text is taken from <ref type="work" corresp="BDLaethe32"/>.</note>
+                <div type="textpart" xml:lang="gez">
+                    <ab>
+                        ሰላም፡ ለአስተረዕዮትከ፡ ውስተ፡ ደብረ፤ ማህው፡ ዘዮም፡ ዘምስለ፡ <sic>ቅብ</sic>
```
Comment (MassimoVilla) [inline new/LIT6703HymnMik.xml:56]: I think it is ለአስተርዕዮትከ፡
Also, in the MS I see `<sic>ወስተ</sic>`, as indicated in https://github.com/BetaMasaheft/Documentation/issues/2182

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea, I just have two minor observations due to the very mediocre handwriting of the additions

**Comment by thea-m** [comment]
Thanks a lot, Massimo!

**Comment by thea-m** [comment]
Can this be merged?

**Review by MassimoVilla — APPROVED** [review]

File: new/LIT6702MashafaKobor.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BDLaethe32"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>ለእመ፡ ኮነ፡ ኮቦር፡ ዮሐንስ፡ በሰኑይ፡ አልቦ፡ ደዌ፡ ለጸብእ፡ አገሳ፡ የሐልቁ፡ ዝናም፡ የሐፅፅ፡ መዓር፡ ይውኅዝ፡
+                        ሰገም፡ ይሰምር፡ ሰናይ፡ ይሰምር፡ ሕፃናት፡ ይሙውቱ፨</ab>
+                </div>
+                <div type="textpart" subtype="explicit">
+                    <ab>ለእመ፡ በእሑድ፡ ሐማም፡ ይበዝኅ፡ እክል፡ ወነግድ፡ ይሠምር፡ ፄው፡ ፀሐፅፅ፡ አልባሰ፡ ሐርስ፡ ዝናም፡
```
Comment (karljonaskarlsson) [inline new/LIT6702MashafaKobor.xml:66]: "የሐፅፅ"?

**Pull request merged**